src/schema/core.yaml

```diff
@@ -953,7 +1016,10 @@ classes:
           tooltip: Provide the GOLD biosample IDs associated with this biosample.
 
       alternative_identifiers:
-        description: Unique identifier for a biosample submitted to additional resources. Matches the entity that has been submitted to NMDC
+        description: >-
```

Comment:
I think this `slot_description` for `alternative_identifiers` is on the wrong class.  Isn't this on `Biosample` class?

Reply:
hm, I don't think I moved the position of alternative_identifiers nor its description in this PR, just reformatted the description of that slot in the classes slot_usage to fit more nicely into a single screen.  will double check it, though!

Reply:
I removed my format change so that this doesn't cause concern.